src/app/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.html

```diff
@@ -65,7 +65,8 @@
 </form>
 <div class="d-flex justify-content-end" mat-dialog-actions>
   <button class="btn secondary-global-button" mat-button (click)="onNoClick()">{{ 'personal-info.pop-up-cancel' | translate }}</button>
-  <button class="btn primary-global-button" mat-button [mat-dialog-close]="addAddressForm.value">
-    {{ 'personal-info.pop-up-add-address' | translate }}
+  <button class="btn primary-global-button"  (click)="addAdress()">
+    {{ data.edit ? ('personal-info.pop-up-save-changes' | translate) 
+    : ('personal-info.pop-up-add-address' | translate ) }}
```

Comment:
it's better to have here **(data.edit ? 'personal-info.pop-up-save-changes' : 'personal-info.pop-up-add-address') | translate **
so we will have only 1 translate pipe
or move logic to ts file